packages/mobile/threads/dominantColors.thread.js

```diff
@@ -0,0 +1,141 @@
+import { self } from 'react-native-threads'
```

Comment:
Wonder if we should throw the code that can be shared between this and the web worker version (packages/web/src/workers/dominantColor.worker.js) into common? 

Could be a TODO for later if it's a lot of work since idt this logic will change very often. But at the very least would be good to add a comment about the existence of the web version/file so that people know to change both if needed.

Reply:
yeah i definitely hear you, the main issue however is that you cant import code from node_modules in threads it looks like... that's why i had to vendor in jimp.min.js instead of installing and importing from node_modules. what you think?